modules/ROOT/pages/testing-writing-your-first-test-case.adoc

```diff
@@ -35,7 +35,7 @@ You must include the following tag in your pom.xml file:
 <parent>
 <groupId>org.mule.extensions</groupId>
 <artifactId>mule-modules-parent</artifactId>
-<version>1.6.0</version>
+<version>1.9.0</version>
```

Comment:
This is incorrect. This would only work for Java 17 compiled connectors and would break all others. In any case, we must direct users towards MTF or else we'll have the same migration issue again when we adopt Java 21. 

The java testing framework should be shown as deprecated and for legacy connectors only

Reply:
MTF is not publicly available, these docs are for all customers